modules/julia/LICENSE

```diff
@@ -0,0 +1,42 @@
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
can you please replace it with the following text:

==
Copyright (C) 2020 by Archit Rungta.
The code is a part of OpenCV and is distributed under Apache 2 license.
See https://www.apache.org/licenses/LICENSE-2.0
==

We are switching from BSD to Apache 2 license, especially for the contributions.

Reply:
Should I replace the entire document with this?

Reply:
yes